README.md

```diff
@@ -4,9 +4,15 @@ Galaxykit is a client library for [galaxy\_ng](https://github.com/ansible/galaxy
 
 It's intended to be a low-dependency and low-abstraction client for performing actions on a remote Automation Hub or galaxy\_ng instance.
 
+### Prerequisites
+
+[Ansible](https://github.com/ansible/ansible).
+
+```pip3 install ansible```
+
 ### Installing:
 
-To install the latest development version, run `pip install -e .` in the root of your checked-out copy of this repository.
+To install the latest development version, run `pip3 install ansible galaxykit` in the root of your checked-out copy of this repository.
```

Comment:
```suggestion
To install the latest development version, run `pip3 install -e .` in the root of your checked-out copy of this repository.
```

if you're using a local checkout, pip install -e needs to point at the package root